src/Microsoft.TestPlatform.Common/ExtensionFramework/Utilities/TestExtensionPluginInformation.cs

```diff
@@ -79,6 +79,11 @@ private static string GetExtensionUri(Type testLoggerType)
                 }
             }
 
+            if(string.IsNullOrEmpty(extensionUri))
+            {
+                EqtTrace.Error("Error!! The type \"{0}\" defined in \"{1}\" does not have ExtensionUri attibute.", testLoggerType.ToString(), testLoggerType.Module.Name);
```

Comment:
Remove Error!!

Reply:
EqtTrace check if it is enabled.
Because you calling ToString(), and you can avoid that by checking the condition